provisioner/k8s/README.md

```diff
@@ -1,3 +1,112 @@
-# k8s
+# k8s Provisioner
 
-A concrete Provisioner implementation for "plain" k8s bundles.
+## Summary
+
+The k8s provisioner is a core rukpak provisioner that knows how to interact with `k8s` format bundles. 
+The `k8s` bundles, or plain bundles, are simply container images containing a set of static kubernetes YAML manifests in a given directory.  
+For example, below is an example Dockerfile that builds a plain `k8s` bundle from a directory containing static manifests.  
+```dockerfile
+FROM scratch
+LABEL manifests="/manifests"
+COPY ./manifests /manifests
+```
+
+The `k8s` provisioner is able to unpack a given `k8s` bundle onto a cluster and then instantiate it, making the
+content of the bundle available in the cluster. It does so by reconciling the `Bundle` and `BundleInstance` types that 
+have the `spec.provisionerClassName` field set to `core.rukpak.io/plain`. This field must be set to the correct provisioner name 
+in order for the k8s provisioner to see and interact with the bundle. 
+
+## Use cases
+
+### Install and apply a specific version of a bundle 
+
+The k8s provisioner can install and make available a specific `k8s` bundle available in the cluster. 
+
+Simply create a `Bundle` resource pointing to a specific version of your bundle, and a `BundleInstance` which references that bundle.
+The k8s provisioner will unpack the provided Bundle onto the cluster, and eventually make the content available on the cluster. 
+
+```yaml
+apiVersion: core.rukpak.io/v1alpha1
```

Comment:
Did we test this out with the plain Combo bundle? Maybe it makes sense to replace it into here like with the root README.

Reply:
Ah nevermind, we use it in a local example below :)

Reply:
I guess so, just for consistency 👍